src/Controls/src/Core/Element/Element.cs

```diff
@@ -139,6 +149,7 @@ private protected virtual IList<Element> LogicalChildrenInternalBackingStore
 		public ReadOnlyCollection<Element> LogicalChildren =>
 			new ReadOnlyCollection<Element>(new TemporaryWrapper(LogicalChildrenInternal));
 
+		/// <inheritdoc/>
```

Comment:
It'll be interesting to see if mdoc is smart enough to pick this up, when it's not on the public declaration.